unity/Assets/Scripts/BaseFPSAgentController.cs

```diff
@@ -2641,12 +2656,13 @@ public SimObjPhysics[] VisibleSimObjs(bool forceVisible)
                 return GetAllVisibleSimObjPhysics(m_Camera, maxVisibleDistance);
             }
         }
-        protected SimObjPhysics[] GetAllVisibleSimObjPhysics(Camera agentCamera, float maxDistance)
+        protected SimObjPhysics[] GetAllVisibleSimObjPhysics(Camera agentCamera, float maxDistance, IEnumerable<SimObjPhysics> simObjs = null)
```

Comment:
Can we rename `simObjs` so that it's more informative for including these means?